Documentation/operations/troubleshooting.rst

```diff
@@ -218,7 +218,7 @@ Ensure Hubble is running correctly
 ----------------------------------
 
 To ensure the Hubble client can connect to the Hubble server running inside
-Cilium, you may use the ``hubble status`` command:
+Cilium, you may use the ``hubble status`` command, from a Cilium pod:
```

Comment:
Will conflict with https://github.com/cilium/cilium/pull/15981, let's also wait that one is merged.